src/data-sync/ChecklyDataSyncer.ts

```diff
@@ -65,44 +71,53 @@ export class ChecklyDataSyncer {
   }
 
   private async syncCheckResultsBetter(checkId: string, from: Date, to: Date) {
+    const startedAt = Date.now();
+
     const periodsToSync = await this.getPeriodsToSync(checkId, from, to);
     const chunkedPeriods = periodsToSync.flatMap((period) =>
       this.divideIntoPeriodChunks(period.from, period.to, 60),
     );
 
     for (const period of chunkedPeriods) {
       await this.syncCheckResultsChunk(checkId, period.from, period.to);
+      await this.trackCheckSyncStatus(
+        checkId,
+        checkly.accountId,
+        period.from,
+        period.to,
+      );
     }
+
+    log.info(
+      {
+        duration_ms: Date.now() - startedAt,
+        checkId,
+      },
+      "Check results for check synced",
+    );
   }
 
   private getPeriodsToSync = async (
     checkId: string,
     from: Date,
     to: Date,
   ): Promise<{ from: Date; to: Date }[]> => {
-    const oldestRecord = await postgres("check_results")
-      .select("startedAt")
+    const checkSyncStatus = await postgres<CheckSyncStatus>("check_sync_status")
```

Comment:
i started putting all queries into the src/db folder, would be great if knex code isn't spreading everywhere